.github/workflows/pull-request.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Python 3.10.x
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
```

Comment:
GitHub was giving me a warning about the old version using a deprecated version of NodeJS (version 12), so just bumped this version in order to resolve that issue